dockers/docker-fpm-frr/frr/bgpd/bgpd.main.conf.j2

```diff
@@ -8,13 +8,21 @@
 ! TSA configuration
 !
 ip prefix-list PL_LoopbackV4 permit {{ get_ipv4_loopback_address(LOOPBACK_INTERFACE, "Loopback0") | ip }}/32
+{% if multi_npu()  == true %}
+  ip prefix-list PL_LoopbackV4 permit {{ get_ipv4_loopback_address(LOOPBACK_INTERFACE, "Loopback4096") | ip }}/32
+{% endif %}
 !
 {% if get_ipv6_loopback_address(LOOPBACK_INTERFACE, "Loopback0") != 'None' %}
 ipv6 prefix-list PL_LoopbackV6 permit {{ get_ipv6_loopback_address(LOOPBACK_INTERFACE, "Loopback0") | replace('/128', '/64') | ip_network }}/64
 {% endif %}
+{% if multi_npu() == true %}
+{% if get_ipv6_loopback_address(LOOPBACK_INTERFACE, "Loopback4096") != 'None' %}
+  ipv6 prefix-list PL_LoopbackV6 permit {{ get_ipv6_loopback_address(LOOPBACK_INTERFACE, "Loopback4096") | replace('/128', '/64') | ip_network }}/64
+{% endif %}
+{% endif %}
```

Comment:
Why do you need it here?

Reply:
@pavel-shirshov, I think this might be needed to support TSA for the multi-asic platforms. 
Let me know if my understanding is wrong. I will remove this change

Reply:
Removed in the latest commit